LIPS/lip-3.md

```diff
@@ -0,0 +1,160 @@
+The proposed improvement will require formalizing sub-governance groups:
+
+1. **Node Operators Committee** to start staking limit increase motions.
+2. **Financial Operations Team** to start funds allocation motions (i.e. LEGO and reward programs).
+
+Node Operators Committee should include addresses of active node operators partnering with Lido, the list is easy to look up in the [Node Operators Registry](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/nos/NodeOperatorsRegistry.sol) smart contract. Financial Operations Team should consist of established DAO members' addresses and be represented as a Gnosis Safe Multisig with access to corresponding Easy Track features.
+
+# Specification
+
+To create a supervised feature as flexible and extendable as possible, we propose splitting Easy Track functionality into several contracts rather than building a single contract for it. 
+> The statement below is abstract, please visit [specification.md](https://github.com/lidofinance/easy-track/blob/master/specification.md) for full detailed specification.
+
+## Easy Track workflow and motion life cycle
+
+A group of core smart contracts will implement basic logic and store data related to Easy Track motions, and a series of separate EVMScriptFactory smart contracts will represent all the supported types of Easy Track motions. 
+This architecture aims to support further extension by adding new types of Easy Track motions without replacing the core of the proposed system.
+
+A normal motion life cycle includes:
+- Motion creation. A motion can be started by an address entitled to start the specific Easy Track motion type.
+- The voting period. A preset timeframe (motion duration) when objections can be submitted by the governance token holders. The motion cannot be enacted until this period expires.
+- Motion enactment. If the objections threshold hasn't been exceeded within or after the voting period, the motion can be enacted by anyone. Before enactment, the motion can be canceled by the address that started it.
+
+This is how it works in terms of smart contract interactions:
+1. A motion can be started by calling `createMotion()` function on `EasyTrack` core contract. 
+2. Upon motion creation, `EasyTrack` calls `createEvmScript()` function on the corresponding `EVMScriptFactory` contract passing all the data required to generate a motion enactment EVM script.
+3. `EVMScriptFactory` smart contract generates the script and returns it to the `EasyTrack` core contract.
+4. `EasyTrack` contract conducts the motion according to the motion settings (i.e. motion duration, objections threshold and maximum active motions count). While active, the motion can be canceled at any time by the address that has started it.
+5. As soon as the motion duration expires, it becomes possible to enact the motion. Anyone can enact the motion from that moment on.
```

Comment:
Need a point on what happens if objection threshold is reached

Reply:
Added the point.